components/simplifiedGrid/SimplifiedGrid.css

```diff
@@ -0,0 +1,49 @@
+@import url("/components/image/Image.css");
+@import url("/components/card/Card.css");
+
+.grid-col-2 {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 10px;
+}
+
+.grid-col-3 {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
```

Comment:
You can use `repeat(3,1fr)` if you want.

Reply:
Thanks, making the changes.